testing/jormungandr-testing-utils/resources/explorer/graphql/schema.graphql

```diff
@@ -1,6 +1,146 @@
 """Directs the executor to query only when the field exists."""
 directive @ifdef on FIELD
 
+type Block0Date {
+  block0_date: Int!
+}
+
+enum DiscriminationEnum {
+  PRODUCTION,
+  TEST,
+}
+
+type Discrimination {
+  discrimination: DiscriminationEnum!
+}
+
+enum ConsensusTypeEnum {
+  BFT,
+  GENESISPRAOS,
+}
+
+type ConsensusType {
+  consensus_type: ConsensusTypeEnum!
+}
+
+type SlotsPerEpoch {
+  slots_per_epoch: Int!
+}
+
+type SlotDuration {
+  slot_duration: Int!
+}
+
+type EpochStabilityDepth {
+  epoch_stability_depth: Int!
```

Comment:
Also in general, what do you think is the advantage of these wrappers? It gives you flexibility for adding new fields later, but I'm not really sure it's worth doing it for every thing.

Reply:
The fist reason of adding wrappers for any field for the `enum ConfigParam` type is that for the GraphQL needed that every filed will implement `Object` trait as I described above and all standard types like `u32`, `u16`, `u8` does not implement it, so it is needed to add a wrapper.

Reply:
This is true, but I rather use a `custom scalar` like all the other numeric types in `scalars.rs`. It's slightly more verbose (although it looks like they added a `scalar!` that wasn't there before, and we could also write a custom macro too), but it just looks nicer in the schema.

Reply:
@enzoc4 , unfortunately scalars are also not allowed for the GraphQL `Union` as a field, for the same reasons ((((